src/pages/api/send_mail/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express';
+
+export default async function handler(req: Request, res: Response): Promise<unknown> {
+  // check if every field is filled in correctly
+  const body = req.body;
+  if (!body.emailAddress || !body.subject || !body.body) {
+    res
+      .status(400)
+      .json({ data: 'One of the required email elements was not filled in correctly' });
+    return;
+  }
+  // TODO: send email to the specified email address
```

Comment:
De emailadressen zou ik in de environment vars steken. Zo kunnen we dat momenteel testen met voorlopige emailadressen, en kunnen we dat later bij de launch heel makkelijk aanpassen. Het verzenden van de emails kan met bijv [Twilio SendGrid](https://sendgrid.com/). Ik heb een account gemaakt, kom de creds maar vragen als je ze nodig hebt.